dispatch/core/storage.py

```diff
@@ -9,7 +9,17 @@ def generate_signed_url(path, content_type, expires_after_seconds=60):
 
     obj = os.path.join(settings.GS_LOCATION, path)
 
-    client = storage.Client()
+    if settings.LOCAL_DEV:
+        dirname = os.path.dirname
+        ##############################################################################
+        # Ensure you have a valid service-account json key file and set its path below
+        ##############################################################################
+        key_file = os.path.join('/', 'ubyssey.ca', 'ubyssey.ca', 'gcs-local.json') 
```

Comment:
This should be a configurable setting (e.g. `settings.GCS_CREDENTIALS_FILE`) instead of a hard-coded variable. The above will only work inside the `ubyssey.ca` repo